src/organizations/organizations.spec.ts

```diff
@@ -141,6 +141,21 @@ describe('Organizations', () => {
     });
   });
 
+  describe('getOrganization', () => {
+    it(`requests an Organization`, async () => {
+      const mock = new MockAdapter(axios);
+      mock.onGet().reply(200, {});
```

Comment:
Can we mock out the response so we can test the value returned from `getOrganization`?